android/res/values/colors.xml

```diff
@@ -147,75 +123,16 @@
   <color name="routing_button_pressed_tint">#80FFFFFF</color>
   <color name="routing_intermediate_point">#43A047</color>
 
-
-  <color name="bs_divider_color">@color/divider</color>
-  <color name="bs_dark_divider_color">@color/divider_night</color>
-
-  <!-- Search -->
-  <color name="search_star_dimmed">#1F000000</color>
-
-  <!-- What's new -->
-  <color name="news_marker_active">#8A000000</color>
-  <color name="news_marker_active_night">@color/white_secondary</color>
-  <color name="news_marker_inactive">#1F000000</color>
-  <color name="news_marker_inactive_night">#4CFFFFFF</color>
-
   <!-- Editor -->
-  <color name="bg_editor_add_enabled">@color/base_accent</color>
-  <color name="bg_editor_add_disabled">#0C000000</color>
-  <color name="bg_editor_login_fb">#FF4861A3</color>
-  <color name="bg_editor_login_fb_pressed">#FF405792</color>
   <color name="bg_editor_light">#FFF2F6F6</color>
-  <color name="bg_editor_light_pressed">#3D000000</color>
-
-  <color name="text_editor_add_disabled">#1A000000</color>
-
-  <!-- Brand colors -->
-  <color name="bg_brand_booking">#FF19457D</color>
-  <color name="bg_brand_booking_pressed">#FF1B549E</color>
-  <color name="bg_brand_opentable">#FFDA3743</color>
-  <color name="bg_brand_opentable_pressed">#FFFC5965</color>
-  <color name="tint_brand_lonely_planet">#004990</color>
-  <color name="tint_brand_lonely_planet_night">#B3FFFFFF</color>
-
-  <!-- Rating colors -->
-  <color name="rating_excellent">#43A047</color>
-  <color name="rating_good">#7CB342</color>
-  <color name="rating_normal">#F5B027</color>
-  <color name="rating_bad">#F4511E</color>
-  <color name="rating_horrible">#E53935</color>
-  <color name="rating_none">#FF888A82</color>
-  <color name="rating_coming_soon">#FF1E96F0</color>
-
-  <color name="tips_and_triks_bg_night">@color/dark_gray</color>
-  <color name="tips_and_triks_bg">#B8FFFFFF</color>
-  <color name="tips_and_triks_primary_text">#FFFFFF</color>
-  <color name="tips_and_triks_primary_text_night">#000000</color>
-  <color name="tips_and_triks_secondary_text">@color/white_secondary</color>
-  <color name="tips_and_triks_secondary_text_night">@color/dark_gray</color>
-
-  <color name="turquoise">#D0F6FF</color>
-  <color name="bookmarks_subs_title">#c60b2b1b</color>
-  <color name="bookmarks_subs_title_night">#c6ffffff</color>
-  <color name="bookmarks_subs_subtitle">#c6152b1c</color>
-  <color name="bookmarks_subs_subtitle_night">#c6ffffff</color>
-  <color name="sale_view_bg">#A335F4</color>
-  <color name="sightseeing_subs_sale_view_bg">#E54E35</color>
-  <color name="bg_sightseeing_subs">#F0FCFF</color>
-  <color name="bg_placeholder">#E2E2E2</color>
+  <color name="bg_editor_light_pressed">#3D000000</color>>
```

Comment:
Extra `>` in the end. Please remove

Reply:
Wow, great catch! Thanks! I'm pushing the fix into the master.

Reply:
Damn how is that even compiling?